appveyor.yml

```diff
@@ -1,40 +1,40 @@
 build: false
-shallow_clone: true
+shallow_clone: false
+skip_branch_with_pr: true
```

Comment:
Another nit: I'm not sure this is all that useful. What we tend to do is disable branch builds altogether through the webhook. (Otherwise, even with this option, I think you still get builds on newly-created branches that don't have an open PR, which is a bit wasteful.)